todo-app/src/App.js

```diff
@@ -1,10 +1,98 @@
 import React, { Component } from 'react';
+import Header from './Components/Header'
+import Container from './Components/Container'
+import { LoadFromLocalStorage, SaveToLocalStorage} from './localStorage'
+
+import './App.css';
+import data from './todoList.json';
```

Comment:
as per the last comment from you on my home work i have changed the way of importing the data